### DISCUSSION TITLE
Need Median function as an aggregator function

### DISCUSSION THREAD

**Discussion by RSoodAMAT** [grafana/grafana#39119] [General]
Hi, 

We are using Prometheus to collect metrices , and same is being pushed to grafana for viewing purpose . 
Currently we don't see any aggregator function for median in the regard. I only see average function in this regard.  

Can you provide a median function for the same.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
